### PR TITLE
Update development.md

### PR DESCRIPTION
**Pull request by emanuele-em**

### CODE REVIEW

File: docs/book/start/development.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ cargo install cargo-generate
 ```
 
 And finally to generate bindings for kernel data structures, you must install
-`bpftool`, either from your distribution or building it from
+`bpftool`. This tool is part of the package `linux-tools-common`, `linux-lowlatency-tools-common` or you can build it from
```
Comment (alessandrod) [inline docs/book/start/development.md:38]: These names are unlikely to be the same across distros? Also IIRC linux-tools-common under ubuntu installs a stub that when executed tells you "you need to install linux-tools-<version>" or something like that

Comment (emanuele-em) [reply docs/book/start/development.md]: You're right but it was not clear for me,btw I rewrite it

File: docs/book/start/development.md [hunk 2]
```diff
@@ -34,9 +34,7 @@ To generate the scaffolding for your project, you're going to need
 cargo install cargo-generate
 ```
 
-And finally to generate bindings for kernel data structures, you must install
-`bpftool`, either from your distribution or building it from
-[source](https://github.com/libbpf/bpftool).
+To generate bindings for kernel data structures, you'll need to install bpftool. You can locate this tool as part of a larger tools package in certain distributions (e.g., `linux-tools-common` on Ubuntu), or you can opt to build it from its [source](https://github.com/libbpf/bpftool).
```
Comment (alessandrod) [inline docs/book/start/development.md:37]: I'd say

```
To generate bindings for kernel types you need to install bpftool. You can install it from your
distribution (run `sudo apt install linux-tools-$(uname -r)` under debian based distributions), or
building from [source])(https://github.com/libbpf/bpftool).
```

Comment (emanuele-em) [reply docs/book/start/development.md]: yes! It's perfect!

**Review by tamird — CHANGES_REQUESTED** [review]
I think we'd rather not encode another project's installation instructions. We can link to their docs, but I don't want to repeat non-canonical details.

**Comment by dave-tucker** [comment]
+1 to what Tamir said. Bear in mind on Fedora `dnf install bpftool` works, as is the case in many other distros - Debian included IIRC.

**Comment by emanuele-em** [comment]
No problem for me, it was the only unclear step for me, but most likely the problem is me 😂

**Comment by tamird** [comment]
Are you still hoping to have this merged? If so, it needs a rebase (and please squash those commits).

**Pull request closed without merging**